lib/plugin/routes/public/teams.js

```diff
@@ -594,7 +594,7 @@ exports.register = function (server, options, next) {
         headers,
         query: _.pick(validation.searchTeam, ['query'])
       },
-      response: { schema: swagger.List(swagger.Team).label('FilteredTeams') }
+      response: { schema: swagger.Search(swagger.ShortTeam).label('FilteredTeams') }
```

Comment:
nice, only issue here is dynamically generating can lead to generic model creation in swagger doc, e.g. if we use the same call for user search, not an issue for this merge.